File: PRAK305-2210817210023-Muhammad Shofy Akmal.py
e = int (input ("Masukkan Jumlah Detik : "))
a = int (e / 86400)
b = int (e % 86400)
c = int (b / 3600)
f = int (b % 3600)
d = int (f / 60)
e = int (f % 60)
if (a>0) :
    print("%d hari %.2d:%.2d:%.2d" %(a,c,d,e))
else     :
    print("%.2d:%.2d:%.2d"% (c,d,e))

File: PRAK305-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
int main() {
int a,b,c,d,e,g;
printf("Masukkan Jumlah Detik : ");
scanf("%d", &e);
a = e / 86400;
b = e % 86400;
c = b / 3600;
g = b % 3600;
d = g / 60;
e = g % 60;
if (a>0) {printf("%d hari %.2d:%.2d:%.2d",a,c,d,e);}
else {printf("%.2d:%.2d:%.2d",c,d,e);}
}